src/mdacli/logger.py

```diff
@@ -15,7 +15,7 @@
 
 
 @contextlib.contextmanager
-def setup_logging(logobj, logfile=None, debug=False):
+def setup_logging(logobj, logfile=None, level=logging.INFO):
```

Comment:
Shouldn't the default be `logging.WARN` (as it is also the default in the logging package)?

Reply:
Yes, good point.

Reply:
Should be fixed.